src/finchlite/finch_assembly/interpreter.py

```diff
@@ -334,6 +335,18 @@ def my_func(*args_e):
                                 f"Unrecognized function definition: {func}"
                             )
                 return AssemblyInterpreterModule(self, kernels)
+            case asm.Print(args):
+                if isinstance(args, tuple):
```

Comment:
you know that `args` will be tuple here because the `args` field in `Print` is defined to be type tuple in the AST declaration. This if branch is redundant.